Gruntfile.js

```diff
@@ -56,7 +57,10 @@ module.exports = function(grunt) {
         }
       },
       e2e: {
-        src: ['tests/e2e/**/*.js'],
+        src: [
+          'tests/e2e/*.spec.js',
+          'tests/e2e/**/*.js',
```

Comment:
I like the `.spec.js` convention and Bede has started using it in webapp so stick with it. I agree with Stefan - I thought `tests/e2e/**/*.js` would also pick up your new test but I assume you tried and it didn't work, so no problem adding it.